pom.xml

```diff
@@ -1078,6 +1078,27 @@
                     </reportPlugins>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>animal-sniffer-maven-plugin</artifactId>
+                <version>1.14</version>
```

Comment:
and now that there's a signatures artifact for 1.8 jdk, we should use it everywhere, see https://github.com/mojohaus/animal-sniffer/issues/1